Using excludeBotUsers() on conversation select element does not confrom to Slack API

**First**
This lib is terrific -- it's really sped up iterating with block kit and the it's very intuitive to use. Really well done!

Happy to do a PR, but I haven't oriented to the internals of the lib yet. Let me know if that'd be helpful!

**Describe the Bug**
When I use `excludeBotUsers` on a conversation-select element, Slack rejects the output: 
```js
{
  code: 'slack_webapi_platform_error',
  data: {
    ok: false,
    error: 'invalid_arguments',
    response_metadata: {
        messages: [
          '[ERROR] failed to match all allowed schemas [json-pointer:/view]',
          '[ERROR] invalid additional property: exclude_bot_users [json-pointer:/view/blocks/0/element]'
        ]
    ...
}
```
The issue appears to be that `exclude_bot_users` is included twice. Once in `filters` where it belongs and once on `element` where it does not:
```js
    "blocks": [
        {
            "block_id": "my-channel",
            ...
            "element": {
                "action_id": "abc-channel-select",
                "exclude_bot_users": true, //  <---- THIS IS THE ISSUE
                "type": "conversations_select",
                "filter": {
                    "exclude_bot_users": true
                }
            },
    ],
}
``` 
If I delete `exclude_bot_users` from `element` before sending it to Slack, it works just fine.

**To Reproduce**
Using Bolt, display this modal in response to a shortcut:
```js
async function showModal({ ack, shortcut, client }) {
  await ack();
  const myModal = Modal({ 
    title: 'My Modal',
    callbackId: 'my-modal',
   submit: 'Submit'
  })
  .blocks(
    Blocks.Input()
      .blockId('my-channel')
      .label('Choose channel')
      .element(
        Elements.ConversationSelect({
          actionId: 'abc-channel-select'
        })
        .excludeBotUsers() // works when I comment this out
      )      
  )
  .buildToObject();

  await client.views.open({
    trigger_id: shortcut.trigger_id,
    view: myModal,
  });
}
```

**Expected Behavior**
The output conforms to the Slack API

--- a/src/lib/slack-dto.ts
+++ b/src/lib/slack-dto.ts
@@ -38,8 +38,6 @@ export enum Param {
   initialConversations = 'initial_conversations',
   defaultToCurrentConversation = 'default_to_current_conversation',
   filter = 'filter',
-  excludeExternalSharedChannels = 'exclude_external_shared_channels',
-  excludeBotUsers = 'exclude_bot_users',
   initialConversation = 'initial_conversation',
   initialDate = 'initial_date',
   minQueryLength = 'min_query_length',

--- a/tests/checks/built-filter-property.ts
+++ b/tests/checks/built-filter-property.ts
@@ -1,0 +1,24 @@
+import { MethodTestArgs } from '../test-config-types';
+import { Prop } from '../../src/constants';
+
+export const builtFilterProperty = (params: MethodTestArgs): void => {
+  const {
+    Class,
+    methodName,
+    propSetterPropName,
+    slackDtoParamName,
+    methodArgMock,
+  } = params;
+
+  test(`Building the object with a defined property of '${propSetterPropName}' should include the DTO property of '${slackDtoParamName}' only on the filter object.`, () => {
+    const instance = new Class();
+
+    instance[methodName](methodArgMock);
+
+    const built = instance.build();
+
+    expect(built).not.toHaveProperty(slackDtoParamName);
+    expect(built[Prop.Filter]).toHaveProperty(slackDtoParamName);
+    expect(built[Prop.Filter][slackDtoParamName]).toBeDefined();
+  });
+};

--- a/tests/checks/index.ts
+++ b/tests/checks/index.ts
@@ -3,6 +3,7 @@ export * from './bool-false-property';
 export * from './bool-settable-property';
 export * from './bool-true-property';
 export * from './built-child-build';
+export * from './built-filter-property';
 export * from './call-from-index';
 export * from './configurable-enum-value-property';
 export * from './end-method-call';

--- a/tests/methods/exclude-bot-users.ts
+++ b/tests/methods/exclude-bot-users.ts
@@ -1,8 +1,9 @@
 import { CompositeBuilderClassConfig } from '../test-config-types';
 import { Prop } from '../../src/constants';
 import { methodArgMocks } from '../mocks/method-arg-mocks';
-import { SlackDto } from '../../src/lib';
 import * as checks from '../checks';
+
+const dtoParamName = 'exclude_bot_users';
 
 export const excludeBotUsers = (params: CompositeBuilderClassConfig): void => {
   const config = {
@@ -10,9 +11,10 @@ export const excludeBotUsers = (params: CompositeBuilderClassConfig): void => {
     methodArgMock: methodArgMocks.excludeBotUsers,
     methodName: Prop.ExcludeBotUsers,
     propSetterPropName: Prop.ExcludeBotUsers,
-    slackDtoParamName: SlackDto.mapParam(Prop.Filter),
+    slackDtoParamName: dtoParamName,
   };
 
   checks.boolTrueProperty(config);
   checks.boolSettableProperty(config);
+  checks.builtFilterProperty(config);
 };

--- a/tests/methods/exclude-external-shared-channels.ts
+++ b/tests/methods/exclude-external-shared-channels.ts
@@ -1,8 +1,9 @@
 import { CompositeBuilderClassConfig } from '../test-config-types';
 import { Prop } from '../../src/constants';
 import { methodArgMocks } from '../mocks/method-arg-mocks';
-import { SlackDto } from '../../src/lib';
 import * as checks from '../checks';
+
+const dtoParamName = 'exclude_external_shared_channels';
 
 export const excludeExternalSharedChannels = (params: CompositeBuilderClassConfig): void => {
   const config = {
@@ -10,9 +11,10 @@ export const excludeExternalSharedChannels = (params: CompositeBuilderClassConfi
     methodArgMock: methodArgMocks.excludeExternalSharedChannels,
     methodName: Prop.ExcludeExternalSharedChannels,
     propSetterPropName: Prop.ExcludeExternalSharedChannels,
-    slackDtoParamName: SlackDto.mapParam(Prop.Filter),
+    slackDtoParamName: dtoParamName,
   };
 
   checks.boolTrueProperty(config);
   checks.boolSettableProperty(config);
+  checks.builtFilterProperty(config);
 };

